receiver/windowsperfcountersreceiver/windowsperfcounters_scraper.go

```diff
@@ -19,6 +19,7 @@ package windowsperfcountersreceiver // import "github.com/open-telemetry/opentel
 
 import (
 	"context"
+	"go.opentelemetry.io/collector/receiver/scrapererror"
```

Comment:
Please move this import to the second group. This should solve the linter errors. Same in the test file